<launch>

  <node pkg="lslidar_n301_driver" type="lslidar_n301_driver_node" name="lslidar_n301_driver_node" output="screen">
    <param name="frame_id" value="laser_link"/>
    <param name="device_ip" value="192.168.1.222"/>
    <param name="device_port" value="2368"/>
  </node>

  <node pkg="lslidar_n301_decoder" type="lslidar_n301_decoder_node" name="lslidar_n301_decoder_node" output="screen">
    <param name="child_frame_id" value="laser_link"/>
    <param name="point_num" value="2000"/>
    <param name="angle_disable_min" value="0"/>
    <param name="angle_disable_max" value="0"/>
    <param name="min_range" value="0.3"/>
    <param name="max_range" value="100.0"/>
    <param name="frequency" value="10.0"/>
    <param name="publish_point_cloud" value="false"/>
  </node>
  

  <!--node name="rviz" pkg="rviz" type="rviz" args="-d $(find lslidar_n301_decoder)/launch/lslidar_n301.rviz" output="screen"/-->

</launch>
